.github/workflows/ci.yml

```diff
@@ -8,6 +8,15 @@ on:
     branches: '**'
```

Comment:
one *might* also want to trigger this workflow on PRs...
See for example https://github.com/eclipse-platform/eclipse.platform.ui/blob/04265dec58c0fa26b5fe7360c0a6d56cf8218517/.github/workflows/ci.yml#L6-L10

Reply:
> one might also want to trigger this workflow on PRs...

In a previous discussion with @rgrunber , we agreed that we don't want it at the moment, to avoid the risk of disturbing current activity on JDT, and the failure here proves it was wise to be cautious.
We may discuss enabling it for PRs later if there is a demand for it from JDT contributors; but at the moment Jenkins seems sufficient and reliable enough.

Reply:
Jenkins does already report problems well enough. The ci.yml here is more added so that contributors can have GitHub builds on their forks for free, to validate their changes, before pushing a PR (that would then be verified on Jenkins).

Reply:
Why not upload the results from jenkins on pr as a comment?